### ISSUE TITLE
Show received event invitations

### ISSUE THREAD

**Issue by andrehgdias** [tutao/tutanota#7206]
When receiving an event invitation or calendar event via mail I want that these items appear in the calendar app so that I can decide to attend the meeting.

- We could show them in a separate list and then the user needs to add them explicitly.
- We could also add them already to the calendar app and show them with a different style.

**Comment by Backspaze** [comment]
This seems very similar to #5755. Possible merge?

**Comment by wrdhub** [comment]
@Backspaze, yeah #5755 is a possible solution for this. We're still figuring out what we want to do with the standalone calendar app in mind.
